### ISSUE TITLE
[Progress Circle] a11y review

### ISSUE THREAD

**Issue by Westbrook** [adobe/spectrum-web-components#1274]
### Feedback

❌ 9 of the loaders on the doc page have no label specified.
❌ No label should be an error
❌ - Indeterminate read as 50% (need to file w/ Apple)

### Path forward

1. Label the docs!
2. Investigate TS and lint options in #1170 
3. Is there a bug for this that we can track?

cc @jnurthen

**Comment by jnurthen** [comment]
https://bugs.webkit.org/show_bug.cgi?id=214330
